src/FsAutoComplete/FsAutoComplete.Stdio.fs

```diff
@@ -41,15 +42,16 @@ let main (commands: Commands) (commandQueue: BlockingCollection<Command>) =
                   CoreResponse.InfoRes (sprintf "Started (PID=%i)" (System.Diagnostics.Process.GetCurrentProcess().Id))
                ]
           | Parse (file, kind, lines) ->
-              let! res = commands.Parse file lines 0 None
+              let isSdkScript = Some config.UseSdkScripts
```

Comment:
this is a bit of a misnomer: isSdkScript is used as the default if we encounter a loose .fs file as well, so really the '--use-sdk-scripts' file is a but more broad than may be realized.